interface/patient_file/deleter.php

```diff
@@ -32,7 +32,7 @@
 require_once($GLOBALS['srcdir'].'/sl_eob.inc.php');
 
  $patient     = $_REQUEST['patient'];
- $encounterid = $_REQUEST['encounterid'];
+ $encounterid = intval($_REQUEST['encounterid']);
```

Comment:
encounterid is secure in this script, so no need for this
(unless you know of a vulnerability)

Reply:
The vulnerability I see in this page 
is that manipulating the GET URL might cause unwanted behavior 
here is a link example and a picture to better explain the issue  

openemr/interface/patient_file/deleter.php?patient=&encounterid=%27%3a1%20and%20you%20must%20give%20me%20a%20million%20dollar%20this%20will%20also%20delete%20yore%20profile

resulting in 
![image](https://user-images.githubusercontent.com/20621444/33803230-9072a476-dd93-11e7-9907-7c4eeaea2684.png)

Will appreciate your opinion on this

Reply:
All I am seeing here is that you are adding text?
A vulnerability would be if you were able to sneak in javascript, which you haven't done. The point of cross scripting attacks are to inject javascript. Injecting text, albeit confusing will get nothing for the bad actor.

Reply:
Ok so I will revert the changes